packages/api/src/memberContext.ts

```diff
@@ -585,22 +648,45 @@ export class MemberContext implements MemberContext {
     }
 
     for (const invoice of invoices.nodes) {
-      if (
-        invoice.sentReminderAt &&
-        new Date(
+      if (!invoice.userID) {
+        logger('memberContext').warn('invoice %s does not have an user ID', invoice.id)
+        continue
+      }
+
+      if (invoice.sentReminderAt) {
+        const SHOULD_SEND_REMINDER_AGAIN_DATE = new Date(
```

Comment:
Don't you think this following part duplicated? as in line 452. Can we have a function for this or?